.azure-pipelines.yml

```diff
@@ -20,3 +20,13 @@ jobs:
       displayName: 'Install build environment'
     - powershell: scripts/azure-pipelines/build.ps1
       displayName: 'Build'
+  - job: macOS
+    pool:
+      vmImage: 'macOS-10.13'
```

Comment:
Does the travis build also use macOS 10.13?

Reply:
Yes.